Decred Wallet/AppDelegate.swift

```diff
@@ -46,10 +48,13 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
 
     func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplicationLaunchOptionsKey: Any]?) -> Bool {
         // Thread.sleep(forTimeInterval: 1.0)
+        AppContext.instance.storage = Storage()
+        AppContext.instance.walletManager = MobilewalletNewLibWallet(NSHomeDirectory()+"/Documents")
+        AppContext.instance.walletManager?.initLoader()
+        
         if(UserDefaults.standard.bool(forKey: "walletCreated") == true){
```

Comment:
I don't think this is the best way to check if a wallet is created. You can check if the wallet db file exists, that's a perfect way to check if a wallet is created.